RecoLocalTracker/SubCollectionProducers/interface/ClusterMultiplicityFilter.h

```diff
@@ -4,23 +4,26 @@
 #include <string>
 
 #include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDFilter.h"
+#include "FWCore/Framework/interface/stream/EDFilter.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
+#include "DataFormats/Common/interface/DetSetVectorNew.h"
 
 
-class ClusterMultiplicityFilter : public edm::EDFilter {
+class ClusterMultiplicityFilter : public edm::stream::EDFilter<> {
```

Comment:
This could easily be converted to a ```edm::global::EDFilter<>``` which would safe memory and be faster for the framework.